Slambook/index.js

```diff
@@ -10,12 +10,9 @@ document.querySelector('.form').addEventListener('submit', function(event) {
     event.preventDefault(); // Prevent form submission
 
     // Log to console when submit event occurs
-    console.log("I am inside submit");})
-//Write your code above

```

Comment:
It seems there is a missing opening bracket '{' for a function or block of code. Check and correct the missing bracket.